cmd/captplanet/run.go

```diff
@@ -11,12 +11,12 @@ import (
 
 	"storj.io/storj/pkg/cfgstruct"
 	"storj.io/storj/pkg/kademlia"
-	"storj.io/storj/pkg/miniogw"
 	"storj.io/storj/pkg/overlay"
 	psserver "storj.io/storj/pkg/piecestore/rpc/server"
 	"storj.io/storj/pkg/pointerdb"
 	"storj.io/storj/pkg/process"
 	"storj.io/storj/pkg/provider"
+	"storj.io/storj/pkg/uplink"
```

Comment:
okay so shoot, i actually think the `miniogw` package is named right. sorry, this is on me for having a poorly stated ticket.

i was most interested in renaming the gateway tool to be the uplink tool. the uplink tool still has a minio gateway as part of it.